7주차/[Lv.2]_테이블_해시_함수/안혜수.js

```diff
@@ -0,0 +1,14 @@
+function solution(data, col, row_begin, row_end) {
+  const answer = [];
+  col -= 1;
+
+  data.sort((a, b) => {
+    if (a[col] === b[col]) return b[0] - a[0];
+    return a[col] - b[col];
+  });
```

Comment:
```suggestion
  data.sort((a, b) => a[col] === b[col] ? b[0] - a[0] : a[col] - b[col]
```
같은 의미로 위와 같이 축약이 가능합니다!! 🙋🏻‍♂️